examples/zoomable-chart/highlight.js

```diff
@@ -0,0 +1,143 @@
+/* @flow */
+import React from 'react';
+import {ScaleUtils, AbstractSeries} from 'react-vis';
+import window from 'global';
+
+export default class Highlight extends AbstractSeries {
+
+  static get requiresSVG() {
+    return true;
+  }
+
+  static displayName = 'HighlightOverlay';
+  static defaultProps = {
+    allow: 'x',
+    color: 'rgb(77, 182, 172)',
+    opacity: 0.3
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      drawing: false,
+      drawArea: {t: 0, r: 0, b: 0, l: 0},
+      startLoc: 0
+    };
+    this._onMouseUp = () => this.onMouseUp();
+  }
+
+  componentDidMount() {
+    window.addEventListener('mouseup', this._onMouseUp);
```

Comment:
could you put these listeners on g below? it seems dangerous to be stapling stuff to the window? eg what if we have multiple of these at once